src/main/kotlin/io/github/mojira/arisa/modules/RemoveVersionModule.kt

```diff
@@ -17,10 +20,23 @@ class RemoveVersionModule(
                 .map { it.remove }
             assertNotEmpty(removeAddedVersions).bind()
             removeAddedVersions.forEach(::run)
+            if (calledMoreThan5Times(changeLog)) {

```

Comment:
This will be triggered on the sixth attempt; not sure if it's intended though.

Reply:
the >= means it will match at 5 👁️

Reply:
But it's comparing with the amount of existing change log items, which doesn't include the current run (the sixth run).

Reply:
ah, I dont think it matters that much in that case honestly, its just one extra time